main.go

```diff
@@ -27,6 +28,16 @@ func main() {
 	nowFlag := flag.Bool("now", false, "Flag to indicate if the integrator should run immediately (env: INTEGRATOR_NOW, default: false).  If used without cron, integrator will run once and then exit.  If now is not set, \"cron\" must be supplied.")
 	flag.Parse()
 
+	err := os.Mkdir("/etc/integratorlogs", 0755)
```

Comment:
I would prefer that the location be configurable via a flag / env var (see how I support both with other options using `getConfigValue`).  If no option is passed in (via flag or env variable), then it should just default to whatever log normally defaults to (STDOUT/STDERR?).

I suggest this because I don't want stuff writing to `/etc` on my development machine, and in production, folks will probably want to configure it anyway (they may not allow programs to write to `/etc`).